Source/WebKit/UIProcess/DigitalCredentials/WKDigitalCredentialsPicker.mm

```diff
@@ -405,6 +407,18 @@ - (void)handleNSError:(NSError *)error
         break;
     }
 
+    if (_page) {
+        String consoleMessage = exceptionData.message;
+        RetainPtr<NSString> debugDescription = error.userInfo[NSDebugDescriptionErrorKey];
+        if ([debugDescription length] > 0)
```

Comment:
No need to compare to `0`.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/1ab3538a17519577fee477d488cdfc8a203322b1